containers/scripts/deploy.sh

```diff
@@ -80,7 +80,8 @@ prepare_deploy() {
         return 4
     fi
 
-    clean
+    # Bring compose down
+    podman-compose -p osh "$PROFILE" down -v
```

Comment:
You can't use `""` here because the `PROFILE` variable is set only when `docker` is in use:
```
+ podman-compose -p osh '' down -v
podman-compose version: 1.0.6
usage: podman-compose [-h] [-v] [--in-pod in_pod] [--pod-args pod_args]
                      [--env-file env_file] [-f file] [-p PROJECT_NAME]
                      [--podman-path PODMAN_PATH] [--podman-args args]
                      [--podman-pull-args args] [--podman-push-args args]
                      [--podman-build-args args] [--podman-inspect-args args]
                      [--podman-run-args args] [--podman-start-args args]
                      [--podman-stop-args args] [--podman-rm-args args]
                      [--podman-volume-args args] [--no-ansi] [--no-cleanup]
                      [--dry-run]
                      {help,version,wait,systemd,pull,push,build,up,down,ps,run,exec,start,stop,restart,logs,config,port,pause,unpause,kill}
                      ...
podman-compose: error: argument command: invalid choice: '' (choose from 'help', 'version', 'wait', 'systemd', 'pull', 'push', 'build', 'up', 'down', 'ps', 'run', 'exec', 'start', 'stop', 'restart', 'logs', 'config', 'port', 'pause', 'unpause', 'kill')
```

We should do something similar like we already have here:
https://github.com/openscanhub/openscanhub/blob/7eed2865bd48f742dd2af7e9ffff0912a5548f57/containers/scripts/deploy.sh#L95-L97